src/core/withExperiment.js

```diff
@@ -5,37 +5,41 @@ import cookie from 'react-cookie';
 
 import { getDisplayName } from 'core/utils';
 
-export type WithExperimentInjectedProps = {|
+export type withExperimentInjectedProps = {|
```

Comment:
not sure about changing this one but I get why you did it.

Reply:
lol..should I change this back?

Reply:
nm..I saw your comment above